packages/integration-react/src/api/ScmAuth.test.ts

```diff
@@ -174,10 +175,15 @@ describe('ScmAuth', () => {
     await expect(
       azureAuth.getCredentials({
         url: 'http://example.com',
-        additionalScope: { customScopes: { azure: ['vso.org'] } },
+        additionalScope: {
+          customScopes: {
+            azure: ['499b84ac-1321-427f-aa17-267ca6975798/vso.org'],
```

Comment:
I've included the GUID for the ADO client ID here, as we should request the scopes like that for now.